airbyte-integrations/bases/base-normalization/Dockerfile

```diff
@@ -1,4 +1,20 @@
-FROM fishtownanalytics/dbt:0.19.0
+FROM fishtownanalytics/dbt:0.19.1
```

Comment:
can you depend on 0.19.0 instead? that's a hard requirement from the dbt-mysql one for the moment I think, so if this works